quattor/types/metadata.pan

```diff
@@ -4,6 +4,10 @@ type structure_branch = {
     "author" ? string
     "name" : string
     "type" : string
+    @{a unique id, e.g. hash of last git commit}
+    "id" ? string
```

Comment:
aquilon inserts the data by writing fields to the host's object template, which works because the broker is in charge of builds.

For example:

```pan
"/metadata/template/branch/name" = "workernode-abc2015";
"/metadata/template/branch/type" = "sandbox";
"/metadata/template/branch/author" = "abc12345";
```